DynCore_GridCompMod.F90

```diff
@@ -7578,7 +7579,14 @@ subroutine Coldstart(gc, import, export, clock, rc)
        bk_is_missing = .true.
     endif
 
-    if (ak_is_missing .or. bk_is_missing) call set_eta(km, ls, ptop, pint, AK, BK)
+    if (ak_is_missing .or. bk_is_missing) then
```

Comment:
This duplicates logic above.   It should be moved into an internal procedure and then just called from the two locations.

Reply:
The interfaces are different. I am not sure how to avoid the logic. Since it is already moved to init(), it is not a big issue.